tests/test_merge_streams_block.py

```diff
@@ -0,0 +1,129 @@
+from collections import defaultdict
+from time import sleep
+from unittest.mock import MagicMock
+
+from nio.common.signal.base import Signal
+from nio.util.support.block_test_case import NIOBlockTestCase
+
+from ..merge_streams_block import MergeStreams
+
+
+class TestMergeStreams(NIOBlockTestCase):
+
+    def setUp(self):
+        super().setUp()
+        # This will keep a list of signals notified for each output
+        self.last_notified = defaultdict(list)
```

Comment:
This is probably in the block template, but we actually don't need to include this last_notified stuff in the block test cases anymore, it's included in the base block test case.